src/test/java/org/folio/ld/dictionary/model/ResourceTest.java

```diff
@@ -0,0 +1,37 @@
+package org.folio.ld.dictionary.model;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.folio.ld.dictionary.PredicateDictionary;
+import org.folio.ld.dictionary.ResourceTypeDictionary;
+import org.junit.jupiter.api.Test;
+
+class ResourceTest {
+
+  @Test
+  void addType_shouldAddType() {
+    var resource = new Resource();
```

Comment:
what about given/when/then approach? It's not so readable without it, right?

Reply:
Done